src/main/java/by/yurachel/web_app/filter/EncodingFilter.java

```diff
@@ -6,15 +6,19 @@
 import jakarta.servlet.ServletResponse;
 import jakarta.servlet.annotation.WebFilter;
 import jakarta.servlet.http.HttpFilter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
 
 @WebFilter(filterName = "encoding", urlPatterns = "/*")
 public class EncodingFilter extends HttpFilter {
+    private static final Logger ROOT_LOGGER = LogManager.getRootLogger();
```

Comment:
Call this field just LOGGER, no need to specidy ROOT prefix (it was on lesson just as an example that we can inject view logger to 1 class)

Logger should be retrieved by calling another method:

_LogManager.getLogger(EncodingFilter.class)_